src/redux/reducer/userReducer.js

```diff
@@ -0,0 +1,12 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const userReducer = createSlice({
+  name: "user",
+  initialState: {
+    userData: {},
+    isLogin: false,
```

Comment:
this key is already defined in the `useOpenModal` hook, what is the source of truth?